common/api/summary/core-frontend.exports.csv

```diff
@@ -382,6 +382,9 @@ public;MarkerImage = HTMLImageElement | HTMLCanvasElement | HTMLVideoElement | I
 public;class MarkerSet
 public;MarkerTextAlign = "left" | "right" | "center" | "start" | "end"
 public;MarkerTextBaseline = "top" | "hanging" | "middle" | "alphabetic" | "ideographic" | "bottom"
+public;MaterialDiffuseProps
```

Comment:
just want to confirm youre ok making these public

Reply:
Yes, the properties were already public.